resources/views/admin/users.blade.php

```diff
@@ -0,0 +1,40 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+  <div class="row">
+    <h3 class="col-sm-10">Categories</h3>
+    <a class="btn col-sm-2 btn-info" href="#">+ Add Category</a>
+  </div>
+
+  <table class="table table-bordered mt-3">
+    <thead class="thead-light text-center">
+      <tr>
+        <th>Title</th>
+        <th>Description</th>
+        <th>Action</th>
+      </tr>
+    </thead>
+
+    @php
+    $categories = \App\Category::get();
```

Comment:
I think it's better that you place this in your Controller as
```php
$categories = Category::all();
```

Then just return it as compact to your blade.

Reply:
I think so too😅